src/api/types.ts

```diff
@@ -79,3 +79,47 @@ export type RawTransactionData = {
 };
 
 export type SwapAPIParams = GetQuoteAPIParams | GetSwapAPIParams;
+
+/**
+ * Note: exported as public Type
+ */
+export type BuildPayTransactionParams = {
+  address: Address; // The address of the wallet paying
+  chainId: number; // The Chain ID of the payment Network (only Base is supported)
+  chargeId: string; // The ID of the Commerce Charge to be paid
```

Comment:
Where can I read more about what's a Commerce Charge? Do we have live docs anywhere?

Reply:
This is probably the page I would point to: https://docs.cdp.coinbase.com/commerce-onchain/docs/crypto-payments for an overview of what a charge is (though not great info for this context, we'll have to update it).

Where do you think we should add this link in the code?

Reply:
Ok let's come back on this one, mostly i am questioning the word `charge`.